Hash/Hash.cs

```diff
@@ -0,0 +1,158 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FunctionHash
+{
+    public class Result
+    {
+        public string hashType { get; set; }
+        public string message { get; set; }
+        public string hashValue { get; set; }
+    }
+    public static class Hash
+    {
+        public static string StrAppend(byte[] result)
+        {
+            StringBuilder strBuilder = new StringBuilder();
+            for (int i = 0; i < result.Length; i++)
+            {
+                strBuilder.Append(result[i].ToString("x2"));

```

Comment:
x2?

Reply:
Standard and required translation to hexadecimal system + lowercase.